### ISSUE TITLE
Documentation Bug? - Graph Traversals do not mention edgeCollectionRestriction as possible option but it appears to function correctly

### ISSUE THREAD

**Issue by iamacup** [arangodb/arangodb#1689]
Hey - really sorry if this is not the palce for this / if I am being crazy here

Basically I wanted to get the parent of some node that was furthest away in the graph (in this case, to make things obvious, the node has some text attribute 'super parent') so after reading the docs, looks like GRAPH_TRAVERSAL_TREE is for me, however i wanted to restrict which edges the graph would actually look at.

Running this AQL: 
`FOR e IN GRAPH_TRAVERSAL_TREE('mygraph', 'questions/29315165609', 'outbound', 'connnection') RETURN e`
I get this output:  http://pastebin.ca/3356210 - i.e pretty much the whole graph as it is connected by various different edges

Running this AQL: 
`FOR e IN GRAPH_TRAVERSAL_TREE('mygraph', 'questions/29315165609', 'outbound', 'connnection', {edgeCollectionRestriction: 'parent_question'}) RETURN e`
i get this output: http://pastebin.ca/3356222 -i.e. exactly what i need.

**TLDR**: both GRAPH_TRAVERSAL_TREE and GRAPH_TRAVERSAL do not mention edgeCollectionRestriction as a possible option in the documentation [here](https://docs.arangodb.com/Aql/GraphOperations.html), but it seems to work just fine

**Comment by dothebart** [comment]
It may be a random side effect that its possible. If you're able to work with the results, you should prefer https://docs.arangodb.com/Aql/GraphTraversals.html

**Comment by iamacup** [comment]
Cool thanks very much! - i took a look, as this is not a support forum i made an SO post about a similar question (filtering based on edge type) as I couldn't find exactly what i wanted in the docs - [linked here](http://stackoverflow.com/questions/35070470/best-method-to-to-filter-graph-edges-by-type-in-aql) in case anyone stumbles accross this incident via google in the distant future.

**Comment by dothebart** [comment]
nice post on SO, will have a look at it later. hint: images work !\[text](link) in md ;-)

**Comment by dothebart** [comment]
Closing this now, as there are no tests ensuring this functionality. We decided not to document (and test) this during the implementation phase, since there was no smart way to ensure its internal integrity, with the probability for it causing badly to debug and detect miss behavior.

**Comment by iamacup** [comment]
Might be cool to have a message somewhere stating that while some parameters from other named graph functions may work, only those documented for a given function are tested and gaurenteed to actually give the results you expect, but this is such a minor thing so...

Regardless, been using ArangoDB for a week now - sexy piece of kit and am deffinately keeping it in the project - Thanks for the help & keep up the amazing work! - also - you can't post images on SO without at least 15 'points' :(

**Comment by dothebart** [comment]
ah, ok didn't know that. happy to have fixed it for you ;-)
